src/main/java/app/attestation/server/AttestationProtocol.java

```diff
@@ -161,10 +161,13 @@
             OS_ENFORCED_FLAGS_OEM_UNLOCK_ALLOWED |
             OS_ENFORCED_FLAGS_SYSTEM_USER;
 
-    private static final String ATTESTATION_APP_PACKAGE_NAME = "app.attestation.auditor";
+    private static final String ATTESTATION_APP_PACKAGE_NAME = BuildConfig.DEBUG ? 
```

Comment:
Can you drop this app name / fingerprint stuff from here? It would make sense to set this up but separately from this PR and in sync with Auditor.